### ISSUE TITLE
item break with elytra

### ISSUE THREAD

**Issue by yellow-pillow** [GC-spigot/battle-pass#106]
breaking elytra does not see the plugin and the quest just does not work...

**Comment by GC-spigot** [comment]
There's literally formats for issues, it's not hard to follow them. Now I don't have examples or any information, I don't know if it's an issue on your end, specific jar or else.

**Comment by GC-spigot** [comment]
Fixed

**Comment by yellow-pillow** [comment]
** Error description**
The elytra breaking task is broken and doesn't work

**Replay**.
try breaking elytra

**Screenshots/video**
https://drive.google.com/file/d/19TKEyoIfYlx2TkBBcTnhhKo_jVV382HP/view?usp=sharing


**Server info (please fill in the following information):**.
- Server version: [ssspigot 1.16.5].
- Plugin version: [latest 3.5.18].

**addition**.
watch the end of the video! despite the fact that the hoe is shown on the task itself about breaking eliters and I showed you the yml file
